Error: Can't resolve ./src/adb

When I try to use `adbkit` in an Electron project I'm working on, I encounter the following errors:

```
 ERROR in C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/index.js
[1]   Module not found: Error: Can't resolve './src/adb' in 'C:\Users\Noah\...\node_modules\@devicefarmer\adbkit'
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/index.js 9:15-35
[1]    @ ../test-plugin/dist/index.js
[1]    @ ./src/main.ts
[1]    @ multi C:/Users/Noah/.../node_modules/electron-webpack/out/electron-main-hmr/main-hmr ./src/main.ts
[1]
[1]   ERROR in C:/Users/Noah/.../node_modules/@devicefarmer/adbkit-logcat/index.js
[1]   Module not found: Error: Can't resolve './src/logcat' in 'C:\Users\Noah\...\node_modules\@devicefarmer\adbkit-logcat'
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit-logcat/index.js 9:15-38
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/lib/adb/client.js
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/lib/adb.js
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/index.js
[1]    @ ../test-plugin/dist/index.js
[1]    @ ./src/main.ts
[1]    @ multi C:/Users/Noah/.../node_modules/electron-webpack/out/electron-main-hmr/main-hmr ./src/main.ts
[1]
[1]   ERROR in C:/Users/Noah/.../node_modules/@devicefarmer/adbkit-monkey/index.js
[1]   Module not found: Error: Can't resolve './src/monkey' in 'C:\Users\Noah\...\node_modules\@devicefarmer\adbkit-monkey'
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit-monkey/index.js 9:15-38
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/lib/adb/client.js
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/lib/adb.js
[1]    @ C:/Users/Noah/.../node_modules/@devicefarmer/adbkit/index.js
[1]    @ ../test-plugin/dist/index.js
[1]    @ ./src/main.ts
[1]    @ multi C:/Users/Noah/.../node_modules/electron-webpack/out/electron-main-hmr/main-hmr ./src/main.ts
```

I'm using electron-webpack to build the binary, but adbkit is included by way of a yarn workspace dependency (I'm using yarn 2 with the `node-modules` nodeLinker option).

I can't figure out why I'm having the issue, and I can't reproduce it with a clean `electron-webpack` project. Rather than focusing on what's causing the issue, it might be easier to try to find a resolution. For example, is it really necessary to cater to coffeescript users at this point? Could we just remove the check and always return the js version?

--- a/index.coffee
+++ b/index.coffee
@@ -1,5 +1,1 @@
-Path = require 'path'
-
-module.exports = switch Path.extname __filename
-  when '.coffee' then require './src/adb'
-  else require './lib/adb'
+module.exports = require './lib/adb'

